FrEee.sln

```diff
@@ -1,20 +1,20 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 15
-VisualStudioVersion = 15.0.28010.2003
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.29911.98
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "FrEee.WinForms", "FrEee.WinForms\FrEee.WinForms.csproj", "{0EE6CCB3-0A56-4353-B9D1-6D2D6C5737A3}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "FrEee", "FrEee\FrEee.csproj", "{E19A4330-CE19-45D6-BCFC-375E065EF041}"

```

Comment:
Why does the FrEee project have the same GUID as the FrEee.Tests project now?

Reply:
There must have been some mix up - I've been creating new projects and moving code around - but I've changed the GUIDs now.